test/unit/logic/vote.js

```diff
@@ -522,6 +522,40 @@ describe('vote', () => {
 		});
 	});
 
+	describe('getBytes', () => {
+		it('should return null for empty asset', () => {
```

Comment:
How much sense does a vote asset have without votes?

An empty asset should throw imo

Reply:
Good point.